docs/how-to/upgrade-cloud.rst

```diff
@@ -2,6 +2,106 @@
 Upgrade a cloud
 ===============
 
+Recommendations for upgrading production clouds
+-----------------------------------------------
+
+1. Be sure to upgrade in a maintenance window
+2. Start with the upgrade on control-plane applications
+3. Don't force the upgrade on non-empty hypervisors
+4. After upgrading the **control-plane**, choose one empty hypervisor to be a `canary node`.
+   After upgrading it, test if it's behaving as expected.
+5. If no issues are found after upgrading the `canary node`, proceed with the upgrade.
```

Comment:
```suggestion
Recommendations and guidelines for upgrading production clouds
-----------------------------------------------

1. Upgrades can be disruptive; perform them during a maintenance window
2. The control-plane must be upgraded before the data-plane. Simply use `cou upgrade` without subcommands to ensure the correct ordering of operations.
3. Forcing the upgrade of non-empty hypervisors will disrupt the connectivity of the VMs they are hosting; live-migrate instances away from hypervisors undergoing an upgrade if possible
4. Use the `hypervisors` subcommand to test the upgrade of a single canary machine before upgrading the rest of the data-plane.
```

Reply:
> The control-plane must be upgraded before the data-plane. Simply use `cou upgrade` 

Did you mean use `cou upgrade control-plane`  ? I guess it makes more sense to the recommendations